addon/index.js

```diff
@@ -361,6 +362,7 @@ exports.onUnload = function(reason) {
     if (store.installedAddons) {
       Object.keys(store.installedAddons).forEach(id => {
         uninstallExperiment({addon_id: id});
+        ExperimentHacks.uninstalled(id)
```

Comment:
The `addonListener` won't get triggered at this point, so we need to do it manually